app/routes/various-routes.ts

```diff
@@ -9,7 +10,7 @@ router.get('/move-to-root-when-helsenorge-is-ready', async (req, res) => {
 });
 
 router.get(`${urls.privacyPolicy}`, (req, res) => {
-  return res.render('pages/privacy-policy');
+  return res.render(`pages/${LANGUAGE}-privacy-policy`);
```

Comment:
Maybe make it readable? Otherwise it says `no-privacy-policy`

```suggestion
  return res.render(`pages/privacy-policy-for-${LANGUAGE}`);
```

Reply:
Haha yeah, good point. Maybe we should make similar changes to the files in `locales/` as well then?

Reply:
We could add a country slug in settings? Like norway, the-netherlands, ...

Reply:
By settings you mean the config? We already have a field called `LANGUAGE` which is what I used above ☺️

Reply:
Also works!

Reply:
Done!